bricks/template/__brick__/{{project_name.snakeCase()}}/README.md

```diff
@@ -17,6 +17,8 @@ Clone the repository
 
 ### Setup
 
+- Dependency Injection (DI), State Management, and Navigating with [get_it](https://pub.dev/packages/get_it), [flutter_riverpod](https://pub.dev/packages/flutter_riverpod), and [go_router](https://pub.dev/packages/go_router).
```

Comment:
Oops, I meant in the root README file instead https://github.com/nimblehq/flutter-templates/blob/ab5559ccd8c79dd819f1f538303d4a4f54b92148/README.md?plain=1#L10